### ISSUE TITLE
Can not run as sudo if installed with nix-env on ubuntu

### ISSUE THREAD

**Issue by PeterHindes** [NixOS/nix#6038]
![image](https://user-images.githubusercontent.com/19994487/152440860-5b9a0243-0106-4408-8636-fced556d0e18.png)

How should I setup nix so that this works?

**Comment by knedlsepp** [comment]
I don't think this issue is specific to nix, but rather to Ubuntu. See for example: https://unix.stackexchange.com/questions/83191/how-to-make-sudo-preserve-path

I personally use `sudo $(which htop)` (TBH this is usually also preceded by a `command not found`, because I keep forgetting about it 😅)

**Comment by tomberek** [comment]
Closing as stale. Re-open if needed.
